test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -0,0 +1,129 @@
+﻿function Write-LogPassThru
+{
+    Param(
+        [Parameter(Mandatory=$true, ValueFromPipeline=$true, Position = 0)]
+        [string] $Message,
+        $Path = "$env:Temp\CodeCoverageRunLogs.txt"
+    )
+
+    $message = "$([Datetime]::Now.ToShortDateString()) - $([datetime]::Now.ToShortTimeString()) : $message"
+
+    if(-not (Test-Path $Path))
+    {
+        Set-Content -Path $Path -Value $Message -Force -PassThru
+    }
+    else
+    {
+        Add-Content -Path $Path -Value $Message -PassThru
+    }
```

Comment:
Fixed.

---
In reply to: [94474364](https://github.com/PowerShell/PowerShell/pull/2918#discussion_r94474364) [](ancestors = 94474364)